webpack.config.client.js

```diff
@@ -73,12 +73,18 @@ module.exports = ({
         }),
       ],
       // specify min/max file sizes for each JS chunk for optimal performance
+      chunkIds: 'named',
       splitChunks: {
         chunks: 'initial',
         automaticNameDelimiter: '-',
         minSize: 184320, // 180kb
         maxSize: 245760, // 240kb
         cacheGroups: {
+          common: {
+            name: true,
+            minChunks: 2,
```

Comment:
if two chunks e.g. MediaAssetPage and PhotoGallery share the same component, this will be split out into a common chunk. If this were three it would need another page type bundle to share a module before it was split out.